R/student_npr_history_plot.R

```diff
@@ -0,0 +1,121 @@
+#' A small multiples plot of a set of students' national percentile rank histories.
+#'
+#' @description This plots a proto-typical example of Edward Tufte's small multiples
+#' concept.  A class of students in one subject and grade is plotted, witch each
+#' student's historical national percentile rank plotted with a simple liear fit.
+#'
+#' The background colors indicate the quartile the assessment earned.
+#'
+#' @param mapvizieR_obj a \code{\link{mapvizieR}} object
+#' @param studentids a set of student ids to subset too.
+#' @param measurementscale the subject of interest
+#'
+#' @return a ggplot2 object
+#'
+#' @export
+#'
+#' @examples
+#'
+#' data("ex_CombinedStudentsBySchool")
+#' data("ex_CombinedAssessmentResults")
+#'
+#' map_mv<-mapvizieR(ex_CombinedAssessmentResults, ex_CombinedStudentsBySchool)
+#'
+#' ids<-ex_CombinedStudentsBySchool %>% filter(
+#'    Grade==8,
+#'    SchoolName=="Mt. Bachelor Middle School",
+#'    TermName=="Spring 2013-2014") %>% select(StudentID) %>%
+#'    unique()
+#'
+#' student_npr_history_plot(
+#'     map_mv,
+#'     studentids = ids[1:80, "StudentID"],
+#'     measurementscale = "Reading")
+
+student_npr_history_plot <- function(mapvizieR_obj,
```

Comment:
i think this is one of the few places where we don't agree on linting.  what does lintr say -- your style, or mine?